pipeline/tasks/publish_python_gapic_tasks.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Tasks for publishing artman output """
+
+import subprocess
+from pipeline.tasks import task_base
+
+
+class DevpiIndexTask(task_base.TaskBase):
+    """ sets the devpi index for publishing: publish_env in
+    config file must be dev, test, staging, or prod"""
```

Comment:
Prefer to reference the variable `_PYTHON_PUB_ENVS` here rather than listing its values to reduce the risk of a stale docstring if we update the variable.